bridge/elrond/client.go

```diff
@@ -384,17 +420,26 @@ func (c *client) Execute(_ context.Context, actionId bridge.ActionId, batch *bri
 	gasLimit := c.gasMapConfig.PerformActionBase + uint64(len(batch.Transactions))*c.gasMapConfig.PerformActionForEach
 	hash, err := c.sendTransaction(builder, gasLimit)
 
+	batchData, err := json.Marshal(batch)
+	if err != nil {
+		c.log.Warn("Elrond: error not critical while serializing transaction", "error", err)
+	}
+
 	if err == nil {
-		c.log.Info("Elrond: Executed action", "action ID", actionId, "hash", hash)
+		c.log.Info("Elrond: Executed action", "actionID", actionId, "batch data", string(batchData), "hash", hash)
 	} else {
-		c.log.Info("Elrond: Execution failed for action", "action ID", actionId, "hash", hash, "error", err.Error())
+		c.log.Info("Elrond: Execution failed for action",
+			"actionID", actionId,
+			"batch data", string(batchData),
+			"hash", hash,
+			"error", err.Error())
```

Comment:
optional: `err` instead of `err.Error()`

Reply:
done